### ISSUE TITLE
webcomponentsjs triggers duplicate XMLHTTPRequest in IE11 with Angular 4

### ISSUE THREAD

**Issue by jaichandra** [webcomponents/webcomponentsjs#881]
I have an app build on Angular 4 and uses Polymer 2 components. The `index.html` is as below:
```
<!doctype html>
<html>

<head>
  <meta charset="utf-8">
  <title>Connect</title>
  <base href="/">
  <meta name="theme-color" content="#009DD9">
  <meta name="viewport" content="width=device-width, initial-scale=1">

  <script src="./assets/bower_components/webcomponentsjs/webcomponents-loader.js"></script>
  <link href="assets/elements.html" rel="import">
</head>

<body>
  <ov-root>Loading...</ov-root>
</body>

</html>
```

`elements.html` contains other polymer element imports. When I run this in IE11 (es5 build made with `polymer build`), the `elements.html` is downloaded twice. This results in multiple `A custom element with name '<>' has already been defined.` errors. It seems to be some race condition between `zone.js` and `webcomponents-lite.js`. This error itself is blowing up the render time exponentially. Can someone please look into this and suggest a workaround?

This can even be replicated in demo of Origami library:
https://github.com/hotforfeature/origami/tree/master/demo

**Comment by jaichandra** [comment]
A little digging shows that below code in `html-imports.js` is getting executed twice in IE11, while Firefox executes it only once.
```
      importer = new Importer()
```
https://github.com/webcomponents/html-imports/blob/master/src/html-imports.js#L735

**Comment by TimvdLippe** [comment]
Thanks for the issue. It has been a great while since this issue was opened. Sadly, this issue misses a live reproduction with the most recent version of the webcomponents polyfill for us to be able to debug and potentially fix. Versions 1 of the polyfills fix a large amount of issues, probably including this one as well.

Please see [the CONTRIBUTING.md](https://github.com/webcomponents/webcomponentsjs/blob/master/CONTRIBUTING.md) for guidelines on filing issues.  You can start from [this jsBin template](http://jsbin.com/birajez/edit?html,console,output) as a starting point and link it in the issue template under "Live Demo".

Feel free to file a new issue following these instructions if you can still reproduce this issue with versions 1 of the polyfills.  Thank you!
